packages/@glimmer/runtime/lib/environment.ts

```diff
@@ -101,9 +102,11 @@ export class EnvironmentImpl implements Environment {
   // Delegate methods and values
   public isInteractive: boolean;
 
+  isArgumentError: typeof isArgumentError;
```

Comment:
I know I made up the original name, but since we are now stashing it on a more general object, we probably want to name it something slightly more specific and less likely to collide/confuse with other meaning, like `isArgumentCaptureError`, `didErrorDuringCapture` or something similar.